db/seeds.rb

```diff
@@ -100,43 +103,44 @@
     email: Faker::Internet.email,
     password: 'password',
     password_confirmation: 'password',
-    role: 0
+    role: :user
   )
-  answer_type = rand(0..2)
-  if answer_type == 2
+
+  answer_type = Question.answer_types.entries.sample
+  if answer_type[0].eql?('choice')
     Question.create(
       question: Faker::Lorem.question(word_count: 3),
-      answer_type: answer_type,
+      answer_type: answer_type[1],
       show_in_list: Faker::Boolean.boolean(true_ratio: 0.7),
       choices: Faker::Lorem.words(number: 3)
     )
   else
     Question.create(
       question: Faker::Lorem.question(word_count: 3),
-      answer_type: answer_type,
+      answer_type: answer_type[1],
       show_in_list: Faker::Boolean.boolean(true_ratio: 0.7)
     )
   end
 end
 
 150.times do
-  eq_type = rand(0..3)
-  if eq_type == 1
+  eq_type = Equipment.equipment_types.entries.sample
+  if eq_type[0].eql?('license')
     Equipment.create(
       name: Faker::Appliance.equipment,
-      equipment_type: eq_type,
-      status: rand(0..2),
-      user_id: rand(2..10),
+      equipment_type: eq_type[1],
+      status: Equipment.statuses.entries.sample[1],
```

Comment:
You can lessen method calls in this line by using `.keys` or `.values` instead of `.entries`.

```
> Equipment.statuses.entries
=> [["deployed", 0], ["stored", 1], ["defective", 2]]
```

```
> Equipment.statuses.values
=> [0, 1, 2]

> Equipment.statuses.keys
=> ["deployed", "stored", "defective"]
```

If you use `.keys`/`.values` instead of `.entries`, you won't need the `[1]` for the `.sample` call.